.changeset/seven-balloons-add.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-liveness": major
+---
+
+**Breaking**: Update liveness component default cdn to use rekognition cdn
```

Comment:
Are we sure we would not want to document the CDN (even for CSP configuration)? Still recommend to get in touch with the team just in case. But aligned in general for a major version bump.

Reply:
I was just re-opening the PR that we previously closed. Still planning to add some documentation updates about what the new default url is